includes/news/v2/mail/featured-stories.php

```diff
@@ -1 +1,43 @@
-<h2>Featured Stories</h2>
+<table width="100%" border="0" align="center" cellpadding="0" bgcolor="#FFF" cellspacing="0" style="width:100%; margin:0; background-color:#FFF;padding-bottom:15px;">
+		<tr>
+<?
+	$count = 0;
+	foreach(get_featured_stories_details() as $details) {
```

Comment:
You can try getting the array index here instead of having to track a separate iterator.

```suggestion
	foreach(get_featured_stories_details() as $index => $details) {
```

It may not work, but it's worth a try to eliminate a couple extra lines of code.